src/tests/ftest/soak/soak.yaml

```diff
@@ -0,0 +1,153 @@
+hosts:
+   test_machines:
+        daos_servers:
+            - boro-29
+        # either compute or partition can be used, not both
+        #compute:
+        #    - boro-30
+        partition: tp
+        gateway: []
+timeout: 1000
+server:
+    server_group: daos_server
+
+pool1:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        setname: daos_server
+    createsize:
+        size: 10000000000
+    createsvc:
+        svcn: 1
+
+job1:
+    name: soak_job_1
+    nodes: 1
+    process_per_node: 1
+    time: 10:00
+    mem: 100
+    jobspec: ior1
+
+job2:
+    name: soak_job_2
+    nodes: 2
+    process_per_node: 2
+    time: 10:00
+    mem: 100
+    jobspec: ior2
+
+job3:
+    name: soak_job_3
+    nodes: 1
+    process_per_node: 1
+    time: 10:00
+    mem: 100
+    jobspec: dmg1
+
+ior1:
+    iteration:
+        iter: 1
```

Comment:
As it's a soak test we can afford to have more number of iterations and make the test run a bit longer.

Reply:
Yes, just want to get it working and can expand run times later